### PR TITLE
added add function implementation

### PR DESCRIPTION
**Pull request by shripada**
Add function that takes two arguments and adds them together to compute the sum.

### CODE REVIEW

**Comment by Mohithrajk** [comment]
yes sir

**Comment by Mohithrajk** [comment]
got it sir

**Comment by ErNikhil11** [comment]
Ok sir

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,5 @@ dist
 
 # TernJS port file
 .tern-port
+
+.history
```
Comment (shripada) [inline .gitignore:106]: What is this .history? Why do we ignore it?

Comment (Mohithrajk) [reply .gitignore:106]: what is TernJS?

Comment (prkarkera) [reply .gitignore:106]: ok

File: math.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+function add(a, b) {
```
Comment (ErNikhil11) [inline math.js:1]: What is function in javascript?

File: .gitignore (hunk 1 above)
Comment (prkarkera) [reply .gitignore:106]: because of . it is not saved

Comment (shripada) [reply .gitignore:106]: This is a hidden file created by VSCode via an extension i installed, so want to ensure this does not get committed.

Comment (prkarkera) [reply .gitignore:106]: it is hidden file

Comment (Mohithrajk) [reply .gitignore:106]: resolved conversation

File: math.js (hunk 2 above)
Comment (ErNikhil11) [inline math.js:1]: What is function in javascript?

Comment (shripada) [reply math.js:1]: That is the fundamental unit of abstracting an operation

**Review by Mohithrajk — COMMENTED** [review]
i approve

**Comment by ErNikhil11** [comment]
I approved.

**Review by prkarkera — APPROVED** [review]

**Review by Mohithrajk — APPROVED** [review]

**Review by ErNikhil11 — APPROVED** [review]

**Pull request merged**